### PR TITLE
Add max files in rotation

### PR DESCRIPTION
**Pull request by Chida82**
To increase the number of files for the log. If you have more that 120 connection like me, in the log there is only a few hours. If you want you can add a parameter to configuration for ad more file for rotation.

### CODE REVIEW

File: chia/util/chia_logging.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def initialize_logging(service_name: str, logging_config: Dict, root_path: Path)
         logger.addHandler(handler)
     else:
         logger = logging.getLogger()
-        handler = ConcurrentRotatingFileHandler(log_path, "a", maxBytes=20 * 1024 * 1024, backupCount=7)
+        handler = ConcurrentRotatingFileHandler(log_path, "a", maxBytes=20 * 1024 * 1024, backupCount=logging_config.get("log_maxfilesrotation", 7))
```
Comment (hoffmang9) [inline chia/util/chia_logging.py:32]: This is great and thank you for your PR. We will need one more item - you can't assume that the entry is in config.yaml so please add the default setting we have now if you don't find it in the config so it is fully backwards compatible.

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Thank you!

**Comment by hoffmang9** [comment]
Note the lint failure is not you - it is us and about to be fixed.

File: chia/util/chia_logging.py (hunk 1 above)
Comment (Chida82) [reply chia/util/chia_logging.py:32]: image
It's work if there isn't the key. I try it.
image
If there isn't the default is 7

Comment (Chida82) [reply chia/util/chia_logging.py:32]: I use get of dictionary with a second parameter for default. The syntax is
image

Comment (Chida82) [reply chia/util/chia_logging.py:32]: I think it's backwards compatible. Are you agree?

**Review by hoffmang9 — DISMISSED** [review]
Thanks again.

**Comment by hoffmang9** [comment]
You will need to run python black on chia/util/chia_logging.py and commit the changes before we can merge. One of the lint failures you can ignore but the SuperLinter is accurate.

**Review by hoffmang9 — APPROVED** [review]
Thanks again.

**Pull request merged**